packages/patternfly-4/react-inline-edit-extension/src/components/InlineEdit/utils/utils.js

```diff
@@ -0,0 +1,26 @@
+const requiredElements = ['INPUT', 'BUTTON'];
+let warningCompleted = false;
+
```

Comment:
not sure about this one. I think it might be useful to see a warning when using inline editing without ids

Reply:
nevertheless it is technically possible to use it without ids if there is only one edit component per cell. You just have to pass it to the formatter by yourself.

Reply:
There are problems with that last approach though - you receive a click on td padding as well..